infra/base-images/base-builder/Dockerfile

```diff
@@ -135,7 +135,7 @@ WORKDIR $SRC
 RUN git clone -b stable https://github.com/google/AFL.git afl
 RUN git clone https://github.com/AFLplusplus/AFLplusplus.git aflplusplus && \
```

Comment:
Did you base this on certain branch, i don't see the code to add afl++ in current Dockerfile on master.

Reply:
it is in the afl++ branch (also the target of this PR)

Reply:
ah, didnt know of that. Will let @jonathanmetzman review then.